src/server.h

```diff
@@ -2771,7 +2771,7 @@ robj *createStringObjectFromLongLong(long long value);
 robj *createStringObjectFromLongLongForValue(long long value);
 robj *createStringObjectFromLongLongWithSds(long long value);
 robj *createStringObjectFromLongDouble(long double value, int humanfriendly);
-robj *createQuicklistObject(void);
+robj *createQuicklistObject(int fill, int depth);
```

Comment:
although`quicklistSetOptions` paramter name is depth, inside quicklist it's all `compress`.
```suggestion
robj *createQuicklistObject(int fill, int compress);
```

Reply:
in this case, i like to change it all to compress (for uniformity)